packages/@glimmer/benchmark-env/src/benchmark/on-modifier.ts

```diff
@@ -30,13 +30,13 @@ class OnModifierManager implements ModifierManager<OnModifierState, null> {
   install(state: OnModifierState) {
     const name = valueForRef(state.nameRef);
     const listener = valueForRef(state.listenerRef);
-    cast(state.element, 'ELEMENT').addEventListener(name, listener);
+    castToBrowser(state.element, 'ELEMENT').addEventListener(name, listener);
```

Comment:
Is this removed for `@glimmer/benchmark-env` performance test also?

Reply:
Ah, it may not be, we can check